app/controllers/phases_controller.rb

```diff
@@ -24,7 +24,8 @@ def edit
     else
       render('/phases/edit', locals: { plan: plan, phase: phase, readonly: readonly, 
                                        question_guidance: plan.guidance_by_question_as_hash,
-                                       guidance_groups: guidance_groups })
+                                       guidance_groups: guidance_groups,
+                                       current_tab: params[:r] || 'all-templates' })
```

Comment:
this action is used for displaying a plan's phase, i.e. the answering of a plan. Please, remove changes applied here.